v19.1/managed-authentication.md

```diff
@@ -0,0 +1,30 @@
+---
+title: Authentication
+summary: Learn about the authentication features for Managed CockroachDB clusters.
+toc: true
+build_for: [managed]
+---
+
+Managed CockroachDB uses TLS 1.2 digital certificates for inter-node authentication, [SSL modes](https://www.postgresql.org/docs/11/libpq-ssl.html) for node identity verification, and password authentication for client identity verification.
```

Comment:
This is always a bit weird. Certificates are not for a specific version of TLS. Instead, I think we should say that we use TLS 1.2 for client/node and node/node communication, and digital certificates for node/node authentication.

Reply:
Is this better:

"Managed CockroachDB uses TLS 1.2 for inter-node and client-node communication, digital certificates for inter-node authentication, [SSL modes](https://www.postgresql.org/docs/11/libpq-ssl.html) for node identity verification, and password authentication for client identity verification."